BOJ/2174_로봇_시뮬레이션/WongueShin.py

```diff
@@ -0,0 +1,67 @@
+import sys
+def input():
+    return sys.stdin.readline().rstrip()
+
+def solution():
+    A, B = map(int, input().split(' '))
+    N, M = map(int, input().split(' '))
+    location = []
+    commandList = []
+    direction = [(0, 1), (1, 0), (0, -1), (-1, 0)]
+    for _ in range(N):
+        temp = input().split(' ')
+        if temp[2] == 'N':
+            temp[2] = 0
+        if temp[2] == 'E':
+            temp[2] = 1
+        if temp[2] == 'S':
+            temp[2] = 2
+        if temp[2] == 'W':
+            temp[2] = 3
+        location.append([int(temp[0]), int(temp[1]), temp[2]])
+    for _ in range(M):
+        temp = input().split(' ')
+        commandList.append((int(temp[0]), temp[1], int(temp[2])))
+
+    mat = [[-1 for i in range(A)] for j in range(B)]

```

Comment:
```suggestion
    mat = [[-1 for _ in range(A)] for j in range(B)]

```
i는 따로 사용하지 않는 것 같으니 이렇게 바꿔도 좋을 것 같네요!